README.md

```diff
@@ -41,6 +41,12 @@ Use `npx capnp-es --help` for full usage info.
 
 See [playground](./playground/) for examples and learn more about `.capnp` schema in [language docs](https://capnproto.org/language.html).
 
+---
+
+You can also use `capnp-es` as a [capnp compiler plugin](https://capnproto.org/capnp-tool.html#compiling-schemas).
```

Comment:
I still think the command would be helpful - at least it would be to me to reproduce a test and the the value might be unexpected?